requirements/requirements_tests.txt

```diff
@@ -0,0 +1,173 @@
+#
```

Comment:
is there some way to have hierarchical requirements?  Seems a lot of requirements are just duplicated.